tests/components/view_image.test.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {shallow} from 'enzyme';
+
+import ViewImageModal from 'components/view_image';
+
+describe('components/ViewImageModal', () => {
+    test('should match snapshot', () => {
+        const fileInfo = {
+            id: '{file_id}'
+        };
+
+        const wrapper = shallow(
+            <ViewImageModal
+                show={true}
+                onModalDismissed={jest.fn()}
+                fileInfos={[fileInfo]}
+            />
+        );
+
+        expect(wrapper).toMatchSnapshot();
+    });
+});
```

Comment:
Test for `ViewImageModal` is a bit low. You may find related samples at `tests/components`. I'd suggest to at least cover the following:
- handleNext
- handlePrev
- handleKeyPress
- onModalShown
- onModalHidden
- componentWillReceiveProps
- showImage
- loadImage
- handleImageLoaded
- handleImageProgress
- handleGetPublicLink
- onMouseEnterImage
- onMouseLeaveImage

![screen shot 2017-10-26 at 11 05 40 am](https://user-images.githubusercontent.com/5334504/32033189-aec3fe08-ba3d-11e7-90a5-3d7c868d00b2.png)

![screen shot 2017-10-26 at 11 11 54 am](https://user-images.githubusercontent.com/5334504/32033322-9417250c-ba3e-11e7-8526-a406f922e9c2.png)